evm_loader/utils/neon-accounts.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+##
+## File: neon-accounts.py
+##
+## Prints Ethereum accounts owned by the Neon EVM program.
+## If executed with 'migrate' argument, performs the
+## migration from V1-accounts to the current version.
+
+import base64
+import io
+import os
+import subprocess
+import sys
+
+from solana.rpc.api import Client
+
+SOLANA_URL = os.environ.get("SOLANA_URL", "http://solana:8899")
+EVM_LOADER = os.environ.get("EVM_LOADER", "53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io")
+
+def do_migrate(address):
```

Comment:
Please add type annotations to all python functions

Reply:
OK.